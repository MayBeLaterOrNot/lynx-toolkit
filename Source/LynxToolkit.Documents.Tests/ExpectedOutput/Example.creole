= EasyWiki =
== Introduction ==
=== Objective ===
# Easy to parse
# Easy source code

== Examples ==
=== Paragraphs ===
Lorem ipsum dolor sit amet,
consectetuer adipiscing elit,
sed diam nonummy nibh euismod
tincidunt ut laoreet dolore
magna aliquam erat volutpat.

Ut wisi enim ad minim veniam,
quis nostrud exerci tation
ullamcorper suscipit lobortis
nisl ut aliquip ex ea commodo
consequat.

=== Line breaks ===
Lorem ipsum dolor sit amet,\\
consectetuer adipiscing
elit,\\
sed diam nonummy

=== Horizontal ruler ===
----

=== Emphasized ===
Lorem //ipsum// dolor sit amet

=== Strong ===
Lorem **ipsum** dolor sit amet

=== Links ===
The syntax is
{{[url|text|title]}}\\
[[http://www.google.com|www.google.com]]\\
[[http://www.google.com|google]]\\
[[http://www.google.com|google]]

=== ¤name1¤ Anchor ===
¤name2¤ Anchors are defined as
{{#name1}}\\
[[#name1|Go to
anchor]]

=== Image ===
The syntax is
{{{source|alt|link}}}\\
{{image.png}}\\
{{image.png|alternative
text}}\\
{{image.png|alternative text}}

=== Unordered List ===
- lorem ipsum   dolor sit amet
- consectetuer adipiscing elit
- sed diam nonummy
- nibh euismod tincidunt
- ut laoreet dolore magna aliquam erat volutpat

=== Ordered list ===
# First
# Second
# Third
# Fourth
# Fifth

=== Table ===
||header1||header2||header 3|
|cell 1a|cell 2a|cell 3a|
|cell 1b|cell 2b|cell 3b|

=== Quote ===
///
Line 1 Line 2
///

=== Code ===
{{{
code line 1
code line 2
}}}

Lorem {{ipsum}} dolor sit amet

=== Special characters ===
© Copyright\\
® Registered
trademark\\
™ Trademark\\
3>2 Greater than\\
2<3 Less
than\\
AT&T Ampersand\\
…
Ellipsis\\
↔ → ← Arrows\\
⇔
⇒ ⇐ Double arrows\\
1024×768
or 1024 × 768\\
Long dashes:
– —

=== Smileys ===
:) :(  

=== Symbols ===
(*) (i) (\) (x) (!) (+) (-)
(?) (_) (y) (n) (on) (off) (h)

=== Escaped ===
*Not emphasized*\\
**Not
bold**\\
[Not link](not
link)\\
----\\
### Not
header\\
- not list\\
1. not
ordered list\\
||Not a
table|\\
|Not a cell|  

=== Other ===
==== //Emphasized// title ====
=== Syntax links ===
- [[http://en.wikipedia.org/wiki/Lightweight_markup_language|en.wikipedia.org/wiki/Lightweight_markup_language]]
- [[http://daringfireball.net/projects/markdown|daringfireball.net/projects/markdown]]
- [[http://github.github.com/github-flavored-markdown|github.github.com/github-flavored-markdown]]
- [[https://confluence.atlassian.com/display/BITBUCKET/Using+Wiki+Markup|https://confluence.atlassian.com/display/BITBUCKET/Using+Wiki+Markup]]
- [[http://codeplex.codeplex.com/wikipage?title=CodePlex%20Wiki%20Markup%20Guide|codeplex.codeplex.com/wikipage?title=CodePlex%20Wiki%20Markup%20Guide]]
- [[http://wikicreole.org|wikicreole.org]]
- [[http://en.wikipedia.org/wiki/Creole_%28markup%29|en.wikipedia.org/wiki/Creole_%28markup%29]]
- [[http://code.google.com/p/support/wiki/WikiSyntax|code.google.com/p/support/wiki/WikiSyntax]]
- [[http://www.mediawiki.org/wiki/Help:Contents|www.mediawiki.org/wiki/Help:Contents]]
- [[http://en.wikipedia.org/wiki/Wikipedia:Cheatsheet|en.wikipedia.org/wiki/Wikipedia:Cheatsheet]]
- [[https://confluence.atlassian.com/display/DOC/Confluence+Wiki+Markup|https://confluence.atlassian.com/display/DOC/Confluence+Wiki+Markup]]