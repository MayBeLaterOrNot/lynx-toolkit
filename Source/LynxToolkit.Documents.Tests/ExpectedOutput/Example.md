EasyWiki
========

Introduction
------------

### Objective
1. Easy to parse
2. Easy source code

Examples
--------

### Paragraphs
Lorem ipsum dolor sit amet,
consectetuer adipiscing elit,
sed diam nonummy nibh euismod tincidunt ut
laoreet dolore magna aliquam
erat volutpat.

Ut wisi enim ad minim veniam,
quis nostrud exerci tation
ullamcorper suscipit lobortis nisl ut
aliquip ex ea commodo
consequat.

### Line breaks
Lorem ipsum dolor sit amet,  
consectetuer adipiscing elit,  
sed diam nonummy

### Horizontal ruler

----

### Emphasized
Lorem *ipsum* dolor sit amet

### Strong
Lorem **ipsum** dolor sit amet

### Links
The syntax is `[url|text|title]`  
[www.google.com](http://www.google.com "")  
[google](http://www.google.com "")  
[google](http://www.google.com "search with google")

### #name1  Anchor
#name2  Anchors are defined as `#name1`  
[Go to anchor](#name1 "")

### Image
The syntax is `{source|alt|link}`  
![image.png]  
![image.png](alternative text)  
![image.png](alternative text)

### Unordered List
- lorem ipsum   dolor sit amet
- consectetuer adipiscing elit
- sed diam nonummy
- nibh euismod tincidunt
- ut laoreet dolore magna
aliquam erat volutpat

### Ordered list
1. First
2. Second
3. Third
4. Fourth
5. Fifth

### Table
||header1||  header2  ||  header 3|
|cell 1a|  cell 2a  |  cell 3a|
|cell 1b|  cell 2b  |  cell 3b|

### Quote
> Line 1 Line 2

### Code
```cs
code line 1
code line 2
```

Lorem `ipsum` dolor sit amet

### Special characters
© Copyright  
® Registered trademark  
™ Trademark  
3>2 Greater than  
2<3 Less than  
AT&T Ampersand  
… Ellipsis  
↔ → ← Arrows  
⇔ ⇒ ⇐ Double arrows  
1024×768 or 1024 × 768  
Long dashes: – —

### Smileys
:) :(  

### Symbols
(*) (i) (\) (x) (!) (+) (-) (?) (_) (y) (n) (on) (off) (h)

### Escaped
*Not emphasized*  
**Not bold**  
[Not link](not link)  
----  
### Not header  
- not list  
1. not ordered list  
||Not a table|  
|Not a cell|  

### Other
#### *Emphasized* title
### Syntax links
- [en.wikipedia.org/wiki/Lightweight_markup_language](http://en.wikipedia.org/wiki/Lightweight_markup_language "")
- [daringfireball.net/projects/markdown](http://daringfireball.net/projects/markdown "")
- [github.github.com/github-flavored-markdown](http://github.github.com/github-flavored-markdown "")
- [https://confluence.atlassian.com/display/BITBUCKET/Using+Wiki+Markup](https://confluence.atlassian.com/display/BITBUCKET/Using+Wiki+Markup "")
- [codeplex.codeplex.com/wikipage?title=CodePlex%20Wiki%20Markup%20Guide](http://codeplex.codeplex.com/wikipage?title=CodePlex%20Wiki%20Markup%20Guide "")
- [wikicreole.org](http://wikicreole.org "")
- [en.wikipedia.org/wiki/Creole_%28markup%29](http://en.wikipedia.org/wiki/Creole_%28markup%29 "")
- [code.google.com/p/support/wiki/WikiSyntax](http://code.google.com/p/support/wiki/WikiSyntax "")
- [www.mediawiki.org/wiki/Help:Contents](http://www.mediawiki.org/wiki/Help:Contents "")
- [en.wikipedia.org/wiki/Wikipedia:Cheatsheet](http://en.wikipedia.org/wiki/Wikipedia:Cheatsheet "")
- [https://confluence.atlassian.com/display/DOC/Confluence+Wiki+Markup](https://confluence.atlassian.com/display/DOC/Confluence+Wiki+Markup "")