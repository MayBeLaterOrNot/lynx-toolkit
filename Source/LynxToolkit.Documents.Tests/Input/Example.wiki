@title Lynx Toolkit Document model
@description Example file
@keywords document model, wiki, syntax, example
@syntax owiki

= Lynx Toolkit Document model
== Introduction
=== Document model
The document model is influenced by HTML and FlowDocument.

The document model contains the following block elements

* Paragraph
* Image
* HorizontalRuler
* Table
* OrderedList
* UnorderedList

and inline elements

* Run
* Strong
* Emphasized
* Image
* Hyperlink
* LineBreak

=== Wiki syntax
The "OWiki" syntax was designed to be easy and efficient to parse. 
It was influenced by Markdown, Creole and Confluence, but the syntax is stricter.

=== Output formats
The document model can be formatted to the following formats

* OWiki
* Markdown wiki
* Creole wiki
* Confluence wiki
* Codeplex wiki
* XHtml
* FlowDocument
* OpenXML (Word)

== Examples

=== Headings
Headings are prefixed with `=` characters as in Creole. Note that the headings should not be suffixed by `=`.
The markdown syntax was not chosen because it is more difficult to implement in the parser.

=== Paragraphs
Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam
nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat
volutpat.

Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper
suscipit lobortis nisl ut aliquip ex ea commodo consequat.

=== Line breaks
Two backslashes defines a line break.\\
Lorem ipsum dolor sit amet,\\
consectetuer adipiscing elit,\\
sed diam nonummy

=== Horizontal ruler
Use three or more hyphens to add a horizontal ruler:

---

-----------------------------------------

=== Emphasized (italic) text
Lorem *ipsum* dolor sit amet

=== Strong (bold) text
Lorem **ipsum** dolor sit amet

=== Links
[http://www.google.com]\\
[http://www.google.com|google]  

=== {#name1} Anchors
* Anchor defined {#name2} in text
* [#name1|Go to anchor]

=== Images
{lorem.jpg}  
{lorem.jpg|alternative text}  
[http://www.google.com|{lorem.jpg|alt text}]

=== Unordered List
* consectetuer adipiscing elit
* sed diam nonummy
* Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat.
* Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy

=== Ordered list
# First
# Second
# Third

=== Table
|| header1 ||  header2  ||  header 3 |
| cell A1  |   cell B1  |    cell C1 |
| cell A2  |   cell B2  |    cell C2 |

=== Table with column span
|cell 1|cell2|
|cell 3|^|

=== Table with row span
|cell 1|cell2|
|¨|cell3|

=== Quote
> Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy
> nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat.
>
> Lorem ipsum dolor sit amet,  
> consectetuer adipiscing elit,  
> sed diam nonummy

=== Code (C#)
```cs
code line 1
code line 2
```

=== Code (xml)
```xml
<?xml version="1.0" encoding="UTF-8"?>
<xml a="b">
  <c>d</c>
</xml>
```

=== Included code (xml)
```xml
@include Example.xml
```

=== Inline code
Code in a paragraph: `Math.Sqrt(2)`

=== Equations
Equations are written in [http://en.wikipedia.org/wiki/TeX|TeX] syntax.
Make sure {{file:latex.exe}} and {{file:dvipng.exe}} is in the `PATH` environment variable.
If you install [http://miktex.org|MiKTeX], both programs will be available in your PATH.

$$x^2$$

$$\left( \begin{array}{ccc}
a & b & c \\
d & e & f \\
g & h & i \end{array} \right)$$

$$\nabla \times \mathbf{E} = -\frac{\partial \mathbf{B}} {\partial t}$$

=== Spans

Press {{key:Ctrl}}+{{key:C}} to copy.
Or select {{command:Edit | Copy}} on the menu.

See {{file:ReadMe.txt}} for more information.

=== Style111

[[style111:
| {lorem.jpg|Caption 1.} | {lorem.jpg|Caption 2.} | {lorem.jpg|Caption 3.} |
]]

=== Style12

[[style12:
| {lorem.jpg|Image 1.}\\{lorem.jpg|Image 2.} | Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex ea commodo consequat. Duis autem vel eum iriure dolor in hendrerit in vulputate velit esse molestie consequat, vel illum dolore eu feugiat nulla facilisis at vero eros et accumsan et iusto odio dignissim qui blandit praesent luptatum zzril delenit augue duis dolore te feugait nulla facilisi. Nam liber tempor cum soluta nobis eleifend option congue nihil imperdiet doming id quod mazim placerat facer possim assum. Typi non habent claritatem insitam; est usus legentis in iis qui facit eorum claritatem. Investigationes demonstraverunt lectores legere me lius quod ii legunt saepius. Claritas est etiam processus dynamicus, qui sequitur mutationem consuetudium lectorum. Mirum est notare quam littera gothica, quam nunc putamus parum claram, anteposuerit litterarum formas humanitatis per seacula quarta decima et quinta decima. Eodem modo typi, qui nunc nobis videntur parum clari, fiant sollemnes in futurum. |
]]

=== Style21

[[style21:
| {lorem.jpg|Image 1.}\\{lorem.jpg|Image 2.} | Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex ea commodo consequat. Duis autem vel eum iriure dolor in hendrerit in vulputate velit esse molestie consequat, vel illum dolore eu feugiat nulla facilisis at vero eros et accumsan et iusto odio dignissim qui blandit praesent luptatum zzril delenit augue duis dolore te feugait nulla facilisi. Nam liber tempor cum soluta nobis eleifend option congue nihil imperdiet doming id quod mazim placerat facer possim assum. Typi non habent claritatem insitam; est usus legentis in iis qui facit eorum claritatem. Investigationes demonstraverunt lectores legere me lius quod ii legunt saepius. Claritas est etiam processus dynamicus, qui sequitur mutationem consuetudium lectorum. Mirum est notare quam littera gothica, quam nunc putamus parum claram, anteposuerit litterarum formas humanitatis per seacula quarta decima et quinta decima. Eodem modo typi, qui nunc nobis videntur parum clari, fiant sollemnes in futurum. |
]]

=== Style11

[[style11:
| {lorem.jpg|Image 1.}\\{lorem.jpg|Image 2.} | Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex ea commodo consequat. Duis autem vel eum iriure dolor in hendrerit in vulputate velit esse molestie consequat, vel illum dolore eu feugiat nulla facilisis at vero eros et accumsan et iusto odio dignissim qui blandit praesent luptatum zzril delenit augue duis dolore te feugait nulla facilisi. Nam liber tempor cum soluta nobis eleifend option congue nihil imperdiet doming id quod mazim placerat facer possim assum. Typi non habent claritatem insitam; est usus legentis in iis qui facit eorum claritatem. Investigationes demonstraverunt lectores legere me lius quod ii legunt saepius. Claritas est etiam processus dynamicus, qui sequitur mutationem consuetudium lectorum. Mirum est notare quam littera gothica, quam nunc putamus parum claram, anteposuerit litterarum formas humanitatis per seacula quarta decima et quinta decima. Eodem modo typi, qui nunc nobis videntur parum clari, fiant sollemnes in futurum. |
]]


=== Special characters
| (C) / (c) | © | Copyright |
| (R) / (r) | ® | Registered trademark |
| (TM) / (tm)| ™ | Trademark |
| 3>2 | | Greater than |
| 2<3 | | Less than |
| & | | This is not Ampère's and|
| ... | … | Ellipsis|
|<-> -> <- |↔ → ← |Arrows|
|<=> => <= |⇔ ⇒ ⇐ |Double arrows|
|1024×768 | 1024 × 768  | Cross |
|-- --- | — – | Long dashes |

=== Smileys
| `:)` | :) |
| `:(` | :( |
| `:P` | :P |
| `:D` | :D |
| `;)` | ;) |

=== Symbols
| `(*)` | (*) |
| `(i)` | (i) |
| `(/)` | (/) |
| `(x)` | (x) |
| `(!)` | (!) |
| `(+)` | (+) |
| `(-)` | (-) |
| `(?)` | (?) |
| `(_)` | (_) |
| `(y)` | (y) |
| `(n)` | (n) |
| `(on)` | (on) |
| `(off)` |  (off) |
| `(h)` | (h) |

=== Escaped characters
~\\ Not a new line\\
~*Not emphasized~*\\
~*~*Not bold~*~*\\
~[Not link|not link]\\  
~---\\
~=== Not a header\\
~* Not a list\\
~# Not an ordered list\\
~||Not a table|\\
~|Not a cell|\\

=== Syntax links
* [http://en.wikipedia.org/wiki/Lightweight_markup_language]
* [http://daringfireball.net/projects/markdown]
* [http://github.github.com/github-flavored-markdown]
* [https://confluence.atlassian.com/display/BITBUCKET/Using+Wiki+Markup]
* [http://codeplex.codeplex.com/wikipage?title=CodePlex%20Wiki%20Markup%20Guide]
* [http://wikicreole.org]
* [http://en.wikipedia.org/wiki/Creole_%28markup%29]
* [http://code.google.com/p/support/wiki/WikiSyntax]
* [http://www.mediawiki.org/wiki/Help:Contents]
* [http://en.wikipedia.org/wiki/Wikipedia:Cheatsheet]
* [https://confluence.atlassian.com/display/DOC/Confluence+Wiki+Markup]
* [http://en.wikipedia.org/wiki/Textile_(markup_language)]