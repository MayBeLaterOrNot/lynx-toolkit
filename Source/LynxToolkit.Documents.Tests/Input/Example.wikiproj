<WikiProject>
  <Format>html</Format>
  <Template>ExampleTemplate.html</Template>
  <LocalLinks>{0}.html</LocalLinks>
  <Input>*.wiki</Input>
  <Scc>p4</Scc>
  <Output>.</Output>
  <Variables>
    <Variable Key="version">1.0</Variable>
  </Variables>
  <Defines>
    <Define>ONLINE</Define>
  </Defines>
</WikiProject>