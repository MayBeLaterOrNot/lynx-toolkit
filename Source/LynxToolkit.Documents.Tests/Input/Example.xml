<?xml version="1.0" encoding="UTF-8"?>
<xml a="b">
  <c>d</c>
</xml>
