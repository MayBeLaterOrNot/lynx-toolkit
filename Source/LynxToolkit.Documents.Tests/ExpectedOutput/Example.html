﻿<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <title>EasyWiki example</title>
    <meta name="description" content="This is an example of EasyWiki syntax" />
    <meta name="keywords" content="EasyWiki, wiki, syntax, example" />
  </head>
  <body>
    <h1>EasyWiki</h1>
    <h2>Introduction</h2>
    <h3>Objective</h3>
    <ol>
      <li>Easy to parse</li>
      <li>Easy Source code</li>
    </ol>
    <h2>Examples</h2>
    <h3>Paragraphs</h3>
    <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat.</p>
    <p>Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex ea commodo consequat.</p>
    <h3>Line breaks</h3>
    <p>Lorem ipsum dolor sit amet,<br />consectetuer adipiscing elit,<br />sed diam nonummy</p>
    <h3>Horizontal ruler</h3>
    <hr />
    <h3>Emphasized</h3>
    <p>Lorem <em>ipsum</em> dolor sit amet</p>
    <h3>Strong</h3>
    <p>Lorem <strong>ipsum</strong> dolor sit amet</p>
    <h3>Links</h3>
    <p>The syntax is <code>[url|text|title]</code><br /><a href="http://www.google.com">www.google.com</a><br /><a href="http://www.google.com">google</a><br /><a href="http://www.google.com" title="search with google">google</a></p>
    <h3>
      <a name="name1" /> Anchor</h3>
    <p>
      <a name="name2" /> Anchors are defined as <code>#name1</code><br /><a href="#name1">Go to anchor</a></p>
    <h3>Image</h3>
    <p>The syntax is <code>{Source|alt|link}</code><br /><img src="image.png" alt="" /><br /><img src="image.png" alt="alternative text" /><br /><a href="http://www.google.com"><img src="image.png" alt="alternative text" /></a></p>
    <h3>Unordered List</h3>
    <ul>
      <li>lorem ipsum   dolor sit amet</li>
      <li>consectetuer adipiscing elit</li>
      <li>sed diam nonummy</li>
      <li>nibh euismod tincidunt</li>
      <li>ut laoreet dolore magna aliquam erat volutpat</li>
    </ul>
    <h3>Ordered list</h3>
    <ol>
      <li>First</li>
      <li>Second</li>
      <li>Third</li>
      <li>Fourth</li>
      <li>Fifth</li>
    </ol>
    <h3>Table</h3>
    <table>
      <tr>
        <th>header1</th>
        <th class="c">header2</th>
        <th class="r">header 3</th>
      </tr>
      <tr>
        <td>cell 1a</td>
        <td class="c">cell 2a</td>
        <td class="r">cell 3a</td>
      </tr>
      <tr>
        <td>cell 1b</td>
        <td class="c">cell 2b</td>
        <td class="r">cell 3b</td>
      </tr>
    </table>
    <h3>Quote</h3>
    <blockquote>
      <p>Line 1 Line 2</p>
    </blockquote>
    <h3>Code</h3>
    <pre><code>code line 1
code line 2</code></pre>
    <p>Lorem <code>ipsum</code> dolor sit amet</p>
    <h3>Special characters</h3>
    <p>© Copyright<br />® Registered trademark<br />™ Trademark<br />3&gt;2 Greater than<br />2&lt;3 Less than<br />AT&amp;T Ampersand<br />… Ellipsis<br />↔ → ← Arrows<br />⇔ ⇒ ⇐ Double arrows<br />1024×768 or 1024 × 768<br />Long dashes: – —</p>
    <h3>Smileys</h3>
    <p>
      <img src=":)" alt="" /> <img src=":(" alt="" />  </p>
    <h3>Symbols</h3>
    <p>
      <img src="(*)" alt="" /> <img src="(i)" alt="" /> <img src="(\)" alt="" /> <img src="(x)" alt="" /> <img src="(!)" alt="" /> <img src="(+)" alt="" /> <img src="(-)" alt="" /> <img src="(?)" alt="" /> <img src="(_)" alt="" /> <img src="(y)" alt="" /> <img src="(n)" alt="" /> <img src="(on)" alt="" /> <img src="(off)" alt="" /> <img src="(h)" alt="" /></p>
    <h3>Escaped</h3>
    <p>*Not emphasized*<br />**Not bold**<br />[Not link](not link)<br />----<br />### Not header<br />- not list<br />1. not ordered list<br />||Not a table|<br />|Not a cell|  </p>
    <h3>Other</h3>
    <h4>
      <em>Emphasized</em> title</h4>
    <h3>Syntax links</h3>
    <ul>
      <li>
        <a href="http://en.wikipedia.org/wiki/Lightweight_markup_language">en.wikipedia.org/wiki/Lightweight_markup_language</a>
      </li>
      <li>
        <a href="http://daringfireball.net/projects/markdown">daringfireball.net/projects/markdown</a>
      </li>
      <li>
        <a href="http://github.github.com/github-flavored-markdown">github.github.com/github-flavored-markdown</a>
      </li>
      <li>
        <a href="https://confluence.atlassian.com/display/BITBUCKET/Using+Wiki+Markup">https://confluence.atlassian.com/display/BITBUCKET/Using+Wiki+Markup</a>
      </li>
      <li>
        <a href="http://codeplex.codeplex.com/wikipage?title=CodePlex%20Wiki%20Markup%20Guide">codeplex.codeplex.com/wikipage?title=CodePlex%20Wiki%20Markup%20Guide</a>
      </li>
      <li>
        <a href="http://wikicreole.org">wikicreole.org</a>
      </li>
      <li>
        <a href="http://en.wikipedia.org/wiki/Creole_%28markup%29">en.wikipedia.org/wiki/Creole_%28markup%29</a>
      </li>
      <li>
        <a href="http://code.google.com/p/support/wiki/WikiSyntax">code.google.com/p/support/wiki/WikiSyntax</a>
      </li>
      <li>
        <a href="http://www.mediawiki.org/wiki/Help:Contents">www.mediawiki.org/wiki/Help:Contents</a>
      </li>
      <li>
        <a href="http://en.wikipedia.org/wiki/Wikipedia:Cheatsheet">en.wikipedia.org/wiki/Wikipedia:Cheatsheet</a>
      </li>
      <li>
        <a href="https://confluence.atlassian.com/display/DOC/Confluence+Wiki+Markup">https://confluence.atlassian.com/display/DOC/Confluence+Wiki+Markup</a>
      </li>
    </ul>
  </body>
</html>